endpoints/openrtb2/auction.go

```diff
@@ -1633,3 +1642,48 @@ func checkIfAppRequest(request []byte) (bool, error) {
 	}
 	return false, nil
 }
+
+func (deps *endpointDeps) ExtractFPDForBidders(req *openrtb_ext.RequestWrapper) (map[openrtb_ext.BidderName]*openrtb_ext.ORTB2, []error) {
+	errL := []error{}
+	var resolvedFPD map[openrtb_ext.BidderName]*openrtb_ext.ORTB2
+
+	//If {site,app,user}.ext.data exists, collect it and remove {site,app,user}.ext.data from request
+	globalFpdData, err := firstpartydata.ExtractGlobalFPDData(req)
```

Comment:
Nitpick: `globalFpdData` reads to me as "global first party data data". Could it be shortened to `globalFPD`? Same comment goes for the `ExtractGlobalFPDData` method name.

Reply:
Yes, done